Automattic/wp-calypso
# Data Stores: Remove unused `auth` store

## Proposed Changes

This PR removes the unused Auth store. It hasn't been in use since #42339 landed.

Part of #74399. A follow-up to #73890.

## Testing Instructions

* Verify that the removed store is not in use.
* All checks should be green and Calypso should build well.

## Pre-merge Checklist
